contrib/tracing/log_coins_flushed.py

```diff
@@ -45,12 +49,26 @@
     'ALWAYS'
 ]
 
+# define output data structure in Python
+class Data(ct.Structure):
+    _fields_ = [
+        ("ts", ct.c_long),
+        ("mode", ct.c_uint32),
+        ("coins", ct.c_uint64),
+        ("coins_mem_usage", ct.c_uint64),
+        ("is_flush_prune", ct.c_bool),
+        ("is_full_flush", ct.c_bool)
+    ]
+
```

Comment:
```suggestion
class Data(ctypes.Structure):
    _fields_ = [
        ("ts", ctypes.c_long),
        ("mode", ctypes.c_uint32),
        ("coins", ctypes.c_uint64),
        ("coins_mem_usage", ctypes.c_uint64),
        ("is_flush_prune", ctypes.c_bool),
        ("is_full_flush", ctypes.c_bool)
    ]

```

I'd prefer not to shorten `ctypes` here. As it's 1) not so long and 2) is clearer when reviewing / reading the code.